apps/tailwind-components/pages/DisplayList.story.vue

```diff
@@ -74,3 +66,10 @@ const numberOfDemoItems = ref(3);
     </div>
   </div>
 </template>
+
+<script setup lang="ts">
+const title = ref("My list");
+const columnCount = ref(1);
+const type: Ref<"standard" | "link"> = ref("standard");
```

Comment:
Should we type variables using the `const var: Ref<...>` approach or using `const var = ref<...>()`?